utils/small_map/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "small_map"
+keywords = ["small", "map"]
+categories = ["data-structures"]
+readme = "../../README.md"
+description = "Utility library that add SmallMap data structure"
```

Comment:
Maybe describe the data-structure instead, something like `SmallMap is an inline vec that grows into a heap map past ARRAY_SIZE`